### DISCUSSION TITLE
Welcome to gui Discussions!

### DISCUSSION THREAD

**Discussion by david-vanderson** [david-vanderson/dvui#11] [Announcements]
## Welcome!
This is a way to discuss gui related topics.  Feel free to share ideas, questions, and anything else.

Have a cool gui project or example?  Please share it!

**Comment by iacore** [comment]
Is "gui" above proper noun (refer to this project) or regular noun?

**Reply by david-vanderson** [reply]
Good question, and shows that I really need a better name.  How about "vui"?  It's still easy to type.  A quick search doesn't suggest much conflict.  Then I guess the project would be called Vui, and the functions would all be like ```vui.button()```

**Reply by iacore** [reply]
[No](https://github.com/vlang/ui/)

"easy to type" isn't that much a concern

**Reply by david-vanderson** [reply]
Hmm well since it appears on almost every line I want it to be short.  Your earlier suggestion of "dvui" could work.

**Reply by iacore** [reply]
The project name isn't related to the name in code. afaik you can do `const ui = @import("xxxxxxxx");`

**Reply by david-vanderson** [reply]
That's true, but a large part of making it easy is having the demo code that can be copy-pasted.  I've done it enough myself to be convinced that the internal name should match the external name.  At least in the normal case.  Of course if someone really wanted/needed to they could import it as a different name.

**Reply by iacore** [reply]
I've seen `const rl = @import("raylib");`. When only 1 ui library is used in a project, then you can just call it `ui` no matter the library name.
